### PR TITLE
Feature/product cards

### PR DESCRIPTION
**Pull request by ghislaineguerin**

### CODE REVIEW

File: src/ui-components/lists.scss [hunk 1]
```diff
@@ -214,18 +212,28 @@
...
+    color: #333;
+    padding-bottom: 12px;
+    min-height: 180px;
+    overflow: hidden;
+  }
   .panel {
-    margin: 8px;
-    padding: 8px;
-    border: 1px solid $light-gray;
+    margin: 4px;
```
Comment (ghislaineguerin) [inline src/ui-components/lists.scss]: @RV-Adrian should be replaced by adding panel-body to this component. will be done in future iterations.

**Comment by RV-Adrian** [comment]
@ghislaineguerin 
Looks good overall! A few points to consider:

Height of the Submit-CTA is different.
image

.store.grid-list .grid-list-text { padding:0 } and .submit-product-cta p { margin:0 } could resolve this without extra impact?

The CTA tile uses h3 while the other products use h4. If this is 'by design', that's cool.

I'm not seeing '.grid-list' used anywhere beyond Store now? Can the 'RV Store Only' section be integrated into this SCSS?

File: src/ui-components/lists.scss (hunk 1 above)
Comment (RV-Adrian) [inline src/ui-components/lists.scss]: @ghislaineguerin Sounds good

**Comment by ghislaineguerin** [comment]
@RV-Adrian h3 was a design choice. I have fixed the height and it should be staged now. 
As we've discussed the component should be fixed in future card. Please take a look again.

**Comment by RV-Adrian** [comment]
@ghislaineguerin Great 👍 
Good to go

**Pull request merged**